Modules/posixmodule.c

```diff
@@ -5224,9 +5225,12 @@ parse_file_actions(PyObject *file_actions,
                 {
                     goto fail;
                 }
+                if (PyList_Append(temp_buffer, path)) {
+                    goto fail;
```

Comment:
Hmm, we keep a reference to `path` and need to decrease here?

Reply:
Fixed in 0ee9f25